### PR TITLE
Remove "Usage" section of design.login.gov

### PR DESCRIPTION
**Pull request by juliaelman**
This pull request transitions "Usage" of the design system from living on design.login.gov as a `.md` file to being part of the README. As the user is primarily internal team members, it makes sense to keep this installation and usage information in one place (instead of two).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,178 +0,0 @@
...
-If you’re using Jekyll, a simple plugin can help copy this file during your build process to keep your assets up-to-date. First, add this file to `_plugins/`:
-
-```ruby
-# _plugins/copy_to_destination.rb
-
-module Jekyll
-  module CopyToDestination
-    class CopyGenerator < Generator
-      def generate(site)
-        folders = site.config['copy_to_destination'] || []
```
Comment (zachmargolis) [inline docs/usage.md:39]: can we add this jekyll plugin to the README? I've used it in the handbook

**Review by zachmargolis — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -22,9 +27,12 @@ In development, build the documentation site with assets, watch source files for
 
 ```
 npm start
+
```
Comment (aduth) [inline README.md:30]: Nit: Remove extra line.

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -81,3 +89,94 @@ Before starting, make sure that all changes intended for release should be merge
    - The release version should match the version just published to `npm` (for example, `v2.1.5`).
    - Use the version name as the release title.
    - Use the release notes to link to any important issues or pull requests that were addressed in the release. You may copy this from `CHANGELOG.md`.
+
+## Use with Rails
```
Comment (aduth) [inline README.md:93]: Would it make sense to group the two "Use with/as" headings under a "Usage" heading? Maybe also the Jekyll plugin if we reincorporate it per Zach's comment.

e.g.

```
## Usage

...

### Use with Rails

...

### Use as a JavaScript package

...
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,15 @@
 # Login.gov Design System
 
-## Installation and usage
+This repository contains the code for the Login.gov Design System. The documentation in here includes resources for both how to install locally, but also in various development environments.
 
-See [the documentation](https://design.login.gov/) for installation and usage instructions.
+1. [Installation](#installation)
```
Comment (aduth) [inline README.md:5]: The URL for this includes the "(Locale)" bit.

```suggestion
1. [Installation](#installation-local)
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
+1. [Testing](#testing)
+1. [Deploying documentation updates](#deploying-documentation-updates)
+1. [Publishing a release to `npm`](#publishing-a-release-to-npm)
```
Comment (aduth) [inline README.md:8]: Similar:

```suggestion
1. [Publishing a release to `npm`](#releases)
```

File: docs/index.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ lead: >
           The key to a strong brand <br>is <span class="text-accent-cool">consistency</span>.
         </div>
         <p class="usa-intro">{{ page.lead }}</p>
-        <a href="{{ site.baseurl }}/usage/" class="usa-button usa-button--big">See usage instructions</a>
+        <a href="https://github.com/18F/identity-style-guide#configuring-for-development" class="usa-button usa-button--big">See install instructions on GitHub</a>
```
Comment (aduth) [inline docs/index.md:17]: This section no longer exists:

```suggestion
        <a href="https://github.com/18F/identity-style-guide#installation-local" class="usa-button usa-button--big">See install instructions on GitHub</a>
```

**Review by aduth — APPROVED** [review]

File: docs/usage.md (hunk 1 above)
Comment (juliaelman) [reply docs/usage.md:39]: Yep! Added here 61393d4

File: README.md (hunk 3 above)
Comment (juliaelman) [reply README.md]: I dig it 20933aee75d82c9b2376e713c219c6764af79676

**Comment by juliaelman** [comment]
@aduth updates made. Re-requested review before 🚀 please 😄

**Review by aduth — APPROVED** [review]
Looks good 👍 

("Files changed" look like they include changes from #229, but pretty sure that should merge cleanly)

**Pull request merged**